### PR TITLE
Add Find by ios class chain

### PR DESCRIPTION
**Pull request by rajfidel**
## Change list

This is to add capability to use class chain selectors for Finding elements.
 
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

`            
AppiumWebElement lastSettingsCellItem = MobileAppMgr.CurrentAppDriver.FindElement(MobileBy.IosClassChain("**/XCUIElementTypeCell[-1]"));
string labelAttribute = lastSettingsCellItem.GetAttribute("label");
lastSettingsCellItem.Tap(1, 250);
`

Appium Logs for above code are available [here](https://gist.github.com/rajfidel/cdbe93f1b2ca1e33b6221752cc062c04)

### CODE REVIEW

**Comment by rajfidel** [comment]
@TikhomirovSergey Made requested updates.

File: appium-dotnet-driver/Appium/Enums/MobileSelector.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ public class MobileSelector
         public static readonly string AndroidUIAutomator = "-android uiautomator";
         public static readonly string iOSAutomatoion = "-ios uiautomation";
         public static readonly string iOSPredicateString = "-ios predicate string";
-        public static readonly string WindowsUIAutomation = "-windows uiautomation";
+		public static readonly string iOSClassChain = "-ios class chain";
+		public static readonly string WindowsUIAutomation = "-windows uiautomation";
```
Comment (mykola-mokhnach) [inline appium-dotnet-driver/Appium/Enums/MobileSelector.cs]: Please align new items properly

File: appium-dotnet-driver/Appium/MobileBy.cs [hunk 2]
```diff
@@ -244,4 +246,20 @@ public ByIosNSPredicate(string selector)
         public override string ToString() =>
             $"ByIosNSPredicate({selector})";
     }
+
+	public class ByIosClassChain : MobileBy
+	{
+		/// <summary>
+		/// Initializes a new instance of the <see cref="ByWindowsAutomation"/> class.
```
Comment (mykola-mokhnach) [inline appium-dotnet-driver/Appium/MobileBy.cs]: is this cref correct?

**Comment by rajfidel** [comment]
@mykola-mokhnach Made the fixes.

**Pull request merged**